lighthouse-core/report/html/templates.html

```diff
@@ -366,12 +372,12 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
 
     <div class="lh-tools">
-      <div class="report-icon report-icon--share lh-tools__button" title="Export report">
+      <button class="report-icon report-icon--share lh-tools__button" title="Export report">
```

Comment:
If this stays `div`, would have to 1) use `tabindex="0"` and 2) wire our own key event listener so space/enter would toggle the tools menu.